### PR TITLE
CCD-923: Avoid returning case security_classifications from search API

### PR DESCRIPTION
**Pull request by JamesCollettCGI**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CCD-923

### Change description ###
Avoid returning case security_classifications from API calls.
Updates to 251 files total (4 java + 246 json).

**Does this PR introduce a breaking change?**
```
[x] Yes  --  ** Potential breaking change for any service that uses the removed payload sections. **
[ ] No
```

### CODE REVIEW

**Review by paulridout — DISMISSED** [review]
Changes look OK.

File: src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseDetails.java [hunk 1]
```diff
@@ -222,16 +219,7 @@ public void setLastStateModifiedDate(LocalDateTime lastStateModifiedDate) {
...
-     * @deprecated Will be removed in version 2.x. Use {@link CaseDetails#dataClassification} instead.
-     */
-    @Deprecated
-    @JsonGetter("security_classifications")
-    @ApiModelProperty("Deprecated. Use `data_classification` instead.")
-    public Map<String, JsonNode> getSecurityClassifications() {
-        return dataClassification;
-    }
-
+    @JsonIgnore
```
Comment (danlysiak) [inline src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseDetails.java:222]: We **don't** want to ignore this - we want only the `data_classification`
```suggestion
```

File: src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseDetails.java [hunk 2]
```diff
@@ -73,9 +73,6 @@ public class CaseDetails implements Cloneable {
     @ApiModelProperty("Case data as defined in case type definition. See `docs/api/case-data.md` for data structure.")
     private Map<String, JsonNode> data;
 
-    @JsonProperty("data_classification")
-    @ApiModelProperty("Same structure as `case_data` with classification (`PUBLIC`, `PRIVATE`, `RESTRICTED`) "
-        + "as field's value.")
```
Comment (danlysiak) [inline src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseDetails.java:78]: Can we add these back in please

File: src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseDetails.java (hunk 1 above)
Comment (marvincudjoe) [reply src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseDetails.java]: Done

File: src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseDetails.java (hunk 2 above)
Comment (marvincudjoe) [reply src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseDetails.java]: Done

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Review by paulridout — APPROVED** [review]
Changes look OK.

**Pull request merged**